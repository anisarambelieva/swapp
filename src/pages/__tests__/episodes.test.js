import { render } from '@testing-library/react';

import Episodes from '../episodes.js';

describe('<Episodes/>', () => {
  it('should render Episodes page with correct styles', () => {
    const { container } = render(<Episodes />);

    expect(container).toMatchSnapshot();
  });
});
