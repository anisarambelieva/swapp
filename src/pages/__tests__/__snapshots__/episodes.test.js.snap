// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Episodes/> should render Episodes page with correct styles 1`] = `
.c8 {
  color: #4bd5ee;
  font-weight: bold;
  font-size: 25px;
}

.c7 {
  width: 20rem;
  border-radius: 10px;
  overflow: hidden;
}

.c9 {
  overflow: hidden;
  text-overflow: ellipsis;
  display: -webkit-box;
  -webkit-line-clamp: 10;
  -webkit-box-orient: vertical;
}

.c5 {
  font-weight: bold;
  color: #4bd5ee;
  background-color: #fff;
  border: 0;
}

.c5:hover {
  color: #fff;
  background-color: #4bd5ee;
}

.c1 {
  font-family: starWarsFont;
  font-size: 26px;
  color: #ffe300;
  margin: 0;
  padding-top: 0.2em;
}

.c0 {
  height: 70px;
  background-color: black;
  color: white;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c2 {
  list-style-type: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0;
}

.c3 {
  padding: 0 10px;
}

.c3:last-of-type {
  padding-right: 0;
}

.c4 {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #4bd5ee;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin-top: 30px;
}

<div>
  <div
    class="container"
  >
    <div
      class="c0 row"
    >
      <div
        class="col-md-3"
      >
        <h1
          class="c1"
        >
          SWAPP
        </h1>
      </div>
      <div
        class="col"
        style="justify-content: flex-end; display: flex;"
      >
        <ul
          class="c2"
        >
          <li
            class="c3"
          >
            <a
              class="c4"
              href="#"
            >
              Episodes
            </a>
          </li>
          <li
            class="c3"
          >
            <a
              class="c4"
              href="#"
            >
              Characters
            </a>
          </li>
          <li
            class="c3"
          >
            <button
              class="c5 btn btn-primary"
              type="button"
            >
              Logout
            </button>
          </li>
        </ul>
      </div>
    </div>
    <div
      class="row"
      style="padding-bottom: 30px;"
    >
      <div
        class="c6 col-md-4"
      >
        <div
          class="c7 card"
        >
          <img
            class="card-img"
            src="The_Phantom_Menace.jpg"
            style="height: 300px; object-fit: cover;"
          />
          <div
            class="card-body"
          >
            <div
              class="c8 card-title h5"
            >
              The Phantom Menace
            </div>
            <p
              class="c9 card-text"
            >
              Turmoil has engulfed the Galactic Republic. The taxation of trade routes to outlying star systems is in dispute.Hoping to resolve the matter with a blockade of deadly battleships, the greedy Trade Federation has stopped all shipping to the small planet of Naboo.While the congress of the Republic endlessly debates this alarming chain of events, the Supreme Chancellor has secretly dispatched two Jedi Knights, the guardians of peace and justice in the galaxy, to settle the conflict....
            </p>
          </div>
        </div>
      </div>
      <div
        class="c6 col-md-4"
      >
        <div
          class="c7 card"
        >
          <img
            class="card-img"
            src="Attack_of_the_Clones.jpg"
            style="height: 300px; object-fit: cover;"
          />
          <div
            class="card-body"
          >
            <div
              class="c8 card-title h5"
            >
              Attack of the Clones
            </div>
            <p
              class="c9 card-text"
            >
              There is unrest in the Galactic Senate. Several thousand solar systems have declared their intentions to leave the Republic. This separatist movement, under the leadership of the mysterious Count Dooku, has made it difficult for the limited number of Jedi Knights to maintain peace and order in the galaxy. Senator Amidala, the former Queen of Naboo, is returning to the Galactic Senate to vote on the critical issue of creating an ARMY OF THE REPUBLIC to assist the overwhelmed Jedi....
            </p>
          </div>
        </div>
      </div>
      <div
        class="c6 col-md-4"
      >
        <div
          class="c7 card"
        >
          <img
            class="card-img"
            src="Revenge_of_the_Sith.jpg"
            style="height: 300px; object-fit: cover;"
          />
          <div
            class="card-body"
          >
            <div
              class="c8 card-title h5"
            >
              Revenge of the Sith
            </div>
            <p
              class="c9 card-text"
            >
              War! The Republic is crumbling under attacks by the ruthless Sith Lord, Count Dooku. There are heroes on both sides. Evil is everywhere. In a stunning move, the fiendish droid leader, General Grievous, has swept into the Republic capital and kidnapped Chancellor Palpatine, leader of the Galactic Senate. As the Separatist Droid Army attempts to flee the besieged capital with their valuable hostage, two Jedi Knights lead a desperate mission to rescue the captive Chancellor...
            </p>
          </div>
        </div>
      </div>
      <div
        class="c6 col-md-4"
      >
        <div
          class="c7 card"
        >
          <img
            class="card-img"
            src="A_New_Hope.jpg"
            style="height: 300px; object-fit: cover;"
          />
          <div
            class="card-body"
          >
            <div
              class="c8 card-title h5"
            >
              A New Hope
            </div>
            <p
              class="c9 card-text"
            >
              It is a period of civil war. Rebel spaceships, striking from a hidden base, have won their first victory against the evil Galactic Empire. During the battle, Rebel spies managed to steal secret plans to the Empire’s ultimate weapon, the DEATH STAR, an armoured space station with enough power to destroy an entire planet. Pursued by the Empire’s sinister agents, Princess Leia races home aboard her starship, custodian of the stolen plans that can save her people and restore freedom to the galaxy….
            </p>
          </div>
        </div>
      </div>
      <div
        class="c6 col-md-4"
      >
        <div
          class="c7 card"
        >
          <img
            class="card-img"
            src="The_Empire_Strikes_Back.jpg"
            style="height: 300px; object-fit: cover;"
          />
          <div
            class="card-body"
          >
            <div
              class="c8 card-title h5"
            >
              The Empire Strikes Back
            </div>
            <p
              class="c9 card-text"
            >
              It is a dark time for the Rebellion. Although the Death Star has been destroyed, Imperial troops have driven the Rebel forces from their hidden base and pursued them across the galaxy.Evading the dreaded Imperial Starfleet, a group of freedom fighters led by Luke Skywalker has established a new secret base on the remote ice world of Hoth.The evil lord Darth Vader, obsessed with finding young Skywalker, has dispatched thousands of remote probes into the far reaches of space….
            </p>
          </div>
        </div>
      </div>
      <div
        class="c6 col-md-4"
      >
        <div
          class="c7 card"
        >
          <img
            class="card-img"
            src="Return_of_the_Jedi.jpg"
            style="height: 300px; object-fit: cover;"
          />
          <div
            class="card-body"
          >
            <div
              class="c8 card-title h5"
            >
              Return of the Jedi
            </div>
            <p
              class="c9 card-text"
            >
              Luke Skywalker has returned to his home planet of Tatooine in an attempt to rescue his friend Han Solo from the clutches of the vile gangster Jabba the Hutt.Little does Luke know that the GALACTIC EMPIRE has secretly begun construction on a new armored space station even more powerful than the first dreaded Death Star.When completed, this ultimate weapon will spell certain doom for the small band of rebels struggling to restore freedom to the galaxy...
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
